pyproject.toml

```diff
@@ -1,3 +1,92 @@
 [build-system]
-requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = [
+    "hatchling>=1.10.0",
+    "hatch-nodejs-version>=0.3.0",
+    "jupyterlab~=3.0",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "k3d"
+readme = "README.md"
+requires-python = ">=3.7"
+keywords = [
+    "Jupyter",
+    "JupyterLab",
+    "JupyterLab3",
+]
+classifiers = [
+    "Framework :: Jupyter",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
```

Comment:
```suggestion
```

Reply:
Hi @renefritze! What's the motivation for removing this classifier? The package does support Python 3.7, at least that I recall.

Reply:
Oh, right - I fixed the 3.7 building but didn't push! Let me do that now.

Reply:
It's a duplicate line. I probably should've removed the second instance to make this clear.